scripts/1_create_dialogue.py

```diff
@@ -1,6 +1,11 @@
 """
 This code processes whale vocalization data to generate a structured **dialogue** format from the previously processed **coda sequences**. Here’s a breakdown of what each section of the code does:
 
+**Special coda values**:
+ - 98: silence
+ - 99: change between vocalizations
+ - 100: whether the next row contains ornamentation
```

Comment:
This should say:
 - 100: 'spare'/unalotted click interval that will subsequently be interpreted as ornamentation